model.js

```diff
@@ -32,6 +32,32 @@ exports.selectAllArticles = () => {
     });
 };
 
+exports.selectAllArticlesWithParams = (sort_by, order) => {
+  sort_by = sort_by || "created_at";
+  order = order || "DESC";
```

Comment:
These default parameters can be tidied up slightly by putting them in the parameter to the model, so it'd read `sort_by="created_at` and `order="DESC"`